static/app/views/issueList/savedSearchMenu.tsx

```diff
@@ -1,238 +0,0 @@
-import {Fragment} from 'react';
```

Comment:
yeah probably a good idea to go back to having it all in one file, this was split into two files when we were rolling out the "for review" feature